src/react/deckgl.js

```diff
@@ -67,6 +69,10 @@ export default class DeckGL extends React.Component {
     this._updateLayers(nextProps);
   }
 
+  _onFrameDrawn(canvas) {
```

Comment:
I think you could just pass the callback (i.e. the value of the prop) down to the parent component's prop. No need to define this function to wrap it.